src/posixify.ts

```diff
@@ -0,0 +1,32 @@
+import { isURL } from './isURL';
+import { normalize } from './normalize';
+
+export const posixify = <T extends object = object, A extends Partial<T> = Partial<T>>(obj: T, fallback: A): T => {
```

Comment:
Since we're using `path-browserify`, we don't need to "posix-ify" it. However, it looks like this function is actually URL-ifying the functions, so maybe change the name for clarity?